apps/client/src/features/cuesheet/CuesheetWrapper.tsx

```diff
@@ -69,41 +67,21 @@ export default function CuesheetWrapper() {
         console.error(error);
       }
     },
-    [updateEvent, rundown],
+    [flatRundown, rundownStatus, updateEvent],
   );
 
   const exportHandler = useCallback(
```

Comment:
nit: I'd prefer we kept the handler names `handleSomething`, so `handleExport` in this case